js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchButton.tsx

```diff
@@ -20,7 +20,7 @@ export interface LaunchButtonConfiguration {
   disabled: boolean;
   warning?: React.ReactNode;
   scope?: string;
-  onClick: () => Promise<any>;
+  onClick: (e: React.MouseEvent | KeyboardEvent) => Promise<any>;
```

Comment:
The changes in this file just pass this param through so it's possible to make options that respond to shift-clicking